src/Index/Index.ts

```diff
@@ -13,18 +13,27 @@ const template = html<IndexComponent>`<div class="heading">
       this web page
     </p>
     <div class="buttons">
-      <button class="refresh" @click=${() => triggerAxeCore()}>Refresh</button
-      ><button class="close" @click=${(x) => x.remove()}>Close</button>
+      <button class="refresh" @click=${() => triggerAxeCore()}>Refresh</button>
+      <button @click=${(x) => x.collapse()}>Collapse</button>
+      <button class="close" @click=${(x) => x.remove()}>Close</button>
     </div>
   </div>
   <hr />
-  <slot></slot>`;
+  <slot class="${x => x.collapsed ? 'hide':''}"></slot>`;
 
 @customElement({
   name: "abr-index",
   template,
   styles,
 })
+
 export default class IndexComponent extends FASTElement {
   @attr violationCount = 0;
+  @attr({mode: 'boolean'}) collapsed: boolean = JSON.parse(sessionStorage.getItem('collapsed'));
```

Comment:
We could take advantage of typescript here
```suggestion
  @attr({mode: 'boolean'}) collapsed = !!sessionStorage.getItem('collapsed');
```

Reply:
I put your suggestion in my code and like I said to you earlier, I have an issue with the !! . Even if the value in the sessionStorage is false, with the coalesce the value is always true.

I don't really know what we are doing different but I literally copied the line and it doesn't work, What do we do?

Reply:
You're right, let's leave it this way for now :+1: